### PR TITLE
Updating configuration instructions

### PR DESCRIPTION
**Pull request by arfon**
I've just tried running through the setup and found I couldn't get past step (3) (Docker configuration). Currently running `rake configure:dev_docker` fails with:

```
/opt/rubies/2.1.2/lib/ruby/gems/2.1.0/gems/railties-4.2.1/lib/rails/application/configuration.rb:110:in `database_configuration': Cannot load `Rails.application.database_configuration`:
Could not load database configuration. No such file - ["config/database.yml"] (RuntimeError)
```

I may be missing something but I would expect that any rake task will fail here as the application can't initialize. This PR modifies the setup to break out the `database.yml` config into a separate (earlier) step which solved the issue for me.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,24 +50,23 @@ It's possible to run Panoptes only having to install the `fig_rake` gem. Alterna
 
 #### Usage
 
-1. Clone the repository `git clone https://github.com/zooniverse/Panoptes`.
+0. Clone the repository `git clone https://github.com/zooniverse/Panoptes`.
```
Comment (arfon) [inline README.md]: Changes these to `0` so you don't have to worry about keeping the numbers in order.

**Comment by edpaget** [comment]
Did you mean to submit this to zooniverse/Panoptes?

**Comment by arfon** [comment]
Lol. Yes. How does GitHub work again?

**Comment by edpaget** [comment]
Cool did it work with that change. I thought I had stepped through those instructions on a clean machine before, but I could easily just have been making that up in my imagination.

**Comment by arfon** [comment]
Yep. This fixes things on a blank slate install.

**Comment by edpaget** [comment]
If you want to open a PR against the main repo I'll merge this in then :+1:

**Comment by arfon** [comment]
Closing in favour of https://github.com/zooniverse/Panoptes/pull/912

**Pull request closed without merging**